temporian/core/data/event.py

```diff
@@ -215,6 +215,32 @@ def __rfloordiv__(self, other: Any) -> Event:
             "Only Event and scalar values of type int or float are supported."
         )
 
+    def __gt__(self, other: Any):
+        if isinstance(other, (int, float)):
+            from temporian.core.operators.arithmetic_scalar import (
+                greater_scalar,
+            )
+
+            return greater_scalar(event=self, value=other)
+
+        raise ValueError(
+            f"Cannot compute {type(self)} > {type(other)}. "
+            "Only Event and scalar values of type int or float are supported."
+        )
```

Comment:
Would it be better to raise this in `greater_scalar`?

Reply:
Currently, this function only support scalar comparison. Later, we want to add support for both scalar and non scalar comparison (like truediv above). Keeping the different between scalar and non-scalar looks reasonable in this file, though, it would be great to factorize the code for the different operator, and support swapping operands.

*Adding a todo.

There is also the question to testing the inputs of all the functions in the API. I thing a more systematic solution exist (possibly using decorator; I think we chatted about it :) ). I hope that ultimately, greater_scalar (and all the other API functions) will have such runtime check.